pycontrails/core/met.py

```diff
@@ -2248,6 +2247,26 @@ def shift_longitude(data: XArrayType) -> XArrayType:
     ).sortby("longitude", ascending=True)
 
 
+def shift_longitude_reverse(data: XArrayType) -> XArrayType:
+    """Shift longitude values from [-180, 180) to [0, 360) domain.
+
+    Sorts data by ascending longitude values.
+
+    Parameters
+    ----------
+    data : XArrayType
+        :class:`xr.Dataset` or :class:`xr.DataArray` with longitude dimension
+
+    Returns
+    -------
+    XArrayType
+        :class:`xr.Dataset` or :class:`xr.DataArray` with longitude values on [0, 360).
+    """
+    return data.assign_coords(longitude=(data["longitude"].values + 360.0) % 360.0).sortby(
```

Comment:
If you don't want to use the more general setup I described above, I think the `+ 360.0` term is unnecessary here.